releases/net48/README.md

```diff
@@ -4,6 +4,7 @@ You can learn about the changes made in the .NET Framework 4.8.
 
 - [Download](http://go.microsoft.com/fwlink/?LinkId=2085155)
 - [.NET Framework 4.8 RTM Announcement blog post](https://go.microsoft.com/fwlink/?linkid=2054306)
+- [.NET Framework 4.8 Windows Update Release Blog Post](https://go.microsoft.com/fwlink/?linkid=2100817)
```

Comment:
The link produces a 404 at this point. Is that expected?

Reply:
The Blog is live and you should be able to see the blog post now.